sklearn/cluster/optics_.py

```diff
@@ -197,6 +203,11 @@ class OPTICS(BaseEstimator, ClusterMixin):
     metric_params : dict, optional (default=None)
         Additional keyword arguments for the metric function.
 
+    extract_method : string, optional (default='fancy')
+        The extraction method used to extract clusters using the calculated
+        reachability and ordering. Possible values are "dbscan"
+        and "fancy".
```

Comment:
Is "fancy" the right name??

Reply:
most definitely not, but `auto` can't be the name, since the Xi method is also kinda `auto`. Should we call it __SQLNK__ (first character of authors' last names)?